README.md

```diff
@@ -1,348 +1,44 @@
-# What is RudderStack?
```

Comment:
@ameypv-rudder this repository is the original JS SDK + Autotrack feature. Ideally, we should still have some context around the main SDK.

Reply:
Alright. In that case, I will retain all the content and just update the Autotrack section with the deprecated note.

Reply:
@saikumarrs feedback has been incorporated.